pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["setuptools>=64", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ersilia-client"  
+version = "0.1.0"
+description = "Interact with served Ersilia models through a simple Python client"
+authors = [
+    { name = "Ersilia Open Source Initiative", email = "hello@ersilia.io" }
+]
+readme = "README.md"
+requires-python = ">=3.7"
```

Comment:
Let's try to support Python 3.8 and above, Python 3.7 is slowly being phased out.